ledger/src/shred.rs

```diff
@@ -1605,24 +1614,23 @@ mod tests {
         assert_matches!(bincode::deserialize::<ShredVariant>(&[0b1010_0101]), Err(_));
     }
 
-    #[test_case(false, false, 0b0100_0000)]
-    #[test_case(true, false, 0b0110_0000)]
-    #[test_case(true, true, 0b0111_0000)]
-    fn test_shred_variant_compat_merkle_code(chained: bool, resigned: bool, byte: u8) {
+    #[test_case(false, 0b0110_0000)]
+    #[test_case(true, 0b0111_0000)]
+    fn test_shred_variant_compat_merkle_code(resigned: bool, byte: u8) {
         for proof_size in 0..=15u8 {
             let byte = byte | proof_size;
             assert_eq!(
                 u8::from(ShredVariant::MerkleCode {
                     proof_size,
-                    chained,
+                    chained: true,
```

Comment:
Future cleanup for another PR - we can seemingly drop `chained` from here since it should always be true. We'll just need to make sure we keep the over-the-wire format consistent